src/box/lua/console.lua

```diff
@@ -50,6 +50,10 @@ local output_handlers = { }
 -- puts it inside of a stream.
 local output_eos = { ["yaml"] = '\n...\n', ["lua"] = ';' }
 
+
+-- Each but the last string of multiline command should end with this string.
+local default_multiline_delim = ''
```

Comment:
Nit: A delimiter delimits, a continuation continues. I feel them as different things. I don't feel a continuing a subtype of delimiting.

We already use 'continuation' term. Why don't use it here too and don't introduce the second one? `default_continuation_symbol` looks okay for me.

Reply:
Renamed it.

Reply:
Is it arbitrary string now, right? Not necessary one symbol. So 'symbol' doesn't fit here.

We also agreed to proceed with just `\set continuation on|off`. And now we can greatly simplify the implementation and assume that when the continuation is enabled, it always uses a backslash.

Reply:
I am not sure what simplification you are talking about, but now there is a `continuation_on` local var and `continuation_symbol` as a non-configurable continuation marker.